dist/index.d.ts

```diff
@@ -2447,6 +2447,10 @@ declare var GPUComputePipeline: {
   new (): never;
 };
 
+interface GPUDeviceEventMap {
```

Comment:
nit: `/** @internal */`, and maybe prefix with `__` or something like we do for `__brand`

Reply:
done